ProfileView/src/assets/default-template.ts

```diff
@@ -0,0 +1,36 @@
+export const defaultTemplate = {
+  Name: "Default Template",
+  DatapointCount: 1000,
+  Attributes: [
+    {
+      Version: "0.1",
+      Event: "Quote",
```

Comment:
Quote matches the type of event tracked. May not be used by all clients. If this is to solve for Demo, then its OK. Otherwise it would not work for majority of clients.

Reply:
Or we standardize the way we track firstname and lastname.

Reply:
Well thinking in terms of a universal fallback, what sort of template content is ideal?